tests/model_tests/test_middleware_auth.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from guardian.shortcuts import assign_perm
+import pytest
+# Allow everything in there to access the DB
+pytestmark = pytest.mark.django_db
+
+import logging
+
+from nsot import exc, models
+from nsot.middleware.auth import NsotObjectPermissionsBackend
+
+from .fixtures import admin_user, user, site, transactional_db
+
+def test_object_level_permissions_with_ancestors(site, user):
```

Comment:
Can you please also add a docstring to this test function! :)

Reply:
Done :)